sixtyfps_runtime/rendering_backends/gl/lib.rs

```diff
@@ -876,13 +780,13 @@ impl ItemRenderer for GLItemRenderer {
                     let shadow_image_height = shadow_rect.max_y().ceil() as usize;
 
                     let shadow_image = CachedImage::new_empty_on_gpu(
-                        &self.shared_data.canvas,
+                        &self.canvas,
```

Comment:
There is also a local `canvas` variable. Should that one not be used? Or what's the difference

Reply:
Well spotted, yes, that should be used instead.